src/ui/components/molecules/table-v2/TableV2.module.scss

```diff
@@ -0,0 +1,112 @@
+@import '../../../../globals/ads-variables.scss';
+@import '../../../../globals/fonts.scss';
+
+.table-v2 {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  font-family: Barlow, sans-serif;
+  padding: 16px;
+  width: 100%;
+
+  &.small {
+    gap: 4px;
```

Comment:
px to rem please

Reply:
I have made the correction and updated the units from px to rem.